lib/src/clojure_lsp/db.clj

```diff
@@ -12,14 +12,16 @@
 (def ^:private db-logger-tag "[DB]")
 
 (def initial-db {:documents {}
-                 :processing-changes #{}})
+                 :processing-changes #{}
+                 :dep-graph {}
+                 :file-meta {}})
```

Comment:
I remember when first time checking clojure-lsp one of the things that made me took some time to understand was what was inside the atom...
This change reminded me of that, maybe we could improve this code in the future making that more clear like with a schema or anything that would make that more clear.

Reply:
A schema would be nice, or even just a written description.